choose_your_own_adventure/adventure.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"os"
+	"fmt"
+	"http/template"
+	"encoding/json"
+	"github.com/gopher-training/choose_your_own_adventure"
+
+)
+
+func main() {
+	fileJSON, err := os.Open("cyoa.json")
```

Comment:
defer fileJSON.Close()

Reply:
I'll update it later, in choose_your_adventure branch.